packages/types-dev/objects.d.ts

```diff
@@ -439,6 +438,20 @@ declare global {
                   color: string;
               };
 
+        /**
+         * Object which defines, if the adapter supports receiving messages via sendTo.
+         * Additionally, it defines if specific messages are supported.
+         * If one property is enabled, the object `system.adapter.<adaptername>.<adapterinstance>.messagebox will be created to send messages to the adapter (used for email, pushover, etc...)
+         */
+        interface SupportedMessages {
+            /** If custom messages are supported (same as legacy messagebox) */
+            custom: boolean;
+            /** If notification handling is supported, for information, see https://github.com/foxriver76/ioBroker.notification-manager#requirements-for-messaging-adapters */
+            notifications: boolean;
+            /** If adapter supports signal stopInstance. Use number if you need more than 1000 ms for stop routine. The signal will be sent before stop to the adapter. (used if problems occurred with SIGTERM). */
+            stopInstance: boolean | number;
+        }
```

Comment:
we should also add "getHistory" ... https://github.com/ioBroker/ioBroker.sql/blob/master/io-package.json#L130

But for GetHistory maybe in general note down here that this is new since 5.0 and ideally the old legacy flags should be set in parallel for the time being until adapters are adjusted?!

Reply:
I can add `getHistory`, this was also not in schema yet afaik.

I think the notifications `custom` and `stopInstance`would not need a parallel set, as they are only used by controller. Hence, they would need a js-controller v5 requirement in io-pack instead.